src/boot/efi/secure-boot.c

```diff
@@ -11,3 +11,13 @@ BOOLEAN secure_boot_enabled(void) {
 
         return !EFI_ERROR(err) && secure;
 }
+
+#ifdef SBAT_DISTRO
+static const char sbat[]
+__attribute__((section (".sbat")))
+__attribute__((__used__))
+__attribute__((aligned(512))) = ""
+"sbat,1,SBAT Version,sbat,1,https://github.com/rhboot/shim/blob/main/SBAT.md\n"
+SBAT_PROJECT ",1,systemd EFI boot," SBAT_PROJECT "," PROJECT_VERSION "," PROJECT_URL "\n"
```

Comment:
i don't fully grok the format. but this third field here is supposed to be `vendor_name` aka `human readable vendor name`, right? "systemd EFI boot" sounds like a weird value for that.

Not sure who the "vendor" of sd-boot is but I'd probably suggest something like "The systemd Developers" or "The systemd Project" or so.